opencog/attentionbank/AttentionBank.h

```diff
@@ -85,8 +87,8 @@ class AttentionBank
      * The amount importance funds available in the AttentionBank.
      * Atomic, so that updates don't need a lock.
```

Comment:
See the line above, which says "Atomic, so that updates don't need a lock." ?  Well, you just now mde thme non-atomic, and added a lock.  This means that you should also remove this comment, because its out of sync with the code.

Reply:
I wanted double atomic and after skimming through [this thread](https://stackoverflow.com/questions/30048533/why-isnt-atomic-double-fully-implemented), I opted for the mutex version. I should have removed the comment though.